platforms/corda-ent/configuration/roles/helm_component/templates/node.tpl

```diff
@@ -56,7 +56,15 @@ spec:
         baseDir: /opt/corda
       jarPath: bin
       configPath: etc
-      cordaJarMx: 1524
+      cordaJar:
+        memorySize: 1524
+        unit: M
+      volume:
+        baseDir: /opt/corda/base
+      pod:
+        resources:
+          limits: 2056M
```

Comment:
bit too high. use similar to 1524 for now

Reply:
done changes